### ISSUE TITLE
Version 0.5.4 is not Python 2 compatible

### ISSUE THREAD

**Issue by FredrikAppelros** [uber/vertica-python#89]
The latest release of vertica-python on PyPi breaks compatibility with Python 2. This is not documented, and hopefully, not intentional.

A change made [here](https://github.com/uber/vertica-python/blob/master/vertica_python/vertica/messages/frontend_messages/query.py#L16) raises the following error when a connection attempt is made:
`vertica_python.errors.ConnectionError: str() takes at most 1 argument (2 given)`

I cannot find the change in this repo, but the diff looks like this:
```diff
--- a/vertica-python-0.5.2/vertica_python/vertica/messages/frontend_messages/query.py
+++ b/vertica-python-0.5.4/vertica_python/vertica/messages/frontend_messages/query.py
@@ -1,4 +1,4 @@
-from __future__ import absolute_import
+
 
 from struct import pack
 
@@ -13,7 +13,7 @@ class Query(FrontendMessage):
     def to_bytes(self):
         s = self.query_string
         if isinstance(s, str):
-                s = unicode(s, 'utf-8')
+                s = str(s, 'utf-8')
         encoded = s.encode('utf-8')
         return self.message_string(pack('{0}sx'.format(len(encoded)), encoded))
 ```

**Comment by asafpelegcodes** [comment]
Any plans to fix this?

**Comment by alexjikim** [comment]
sorry about this, i think i accidentally pushed the python3 branch into 0.5.4.  i just released 0.5.5 to pypi which should fix this.
